frontend/public/components/utils/selector-input.jsx

```diff
@@ -24,10 +24,15 @@ export class SelectorInput extends Component {
     };
   }
 
-  componentDidUpdate(prevProps) {
+  componentDidUpdate(prevProps, prevState) {
     if (!_.isEqual(prevProps.tags, this.props.tags)) {
       this.setState({ tags: this.props.tags });
     }
+
+    // Call onValidationChange callback when isInputValid changes
+    if (prevState.isInputValid !== this.state.isInputValid && this.props.onValidationChange) {
+      this.props.onValidationChange(this.state.isInputValid);
```

Comment:
This could moved to its own helper method and then reused in all setState callbacks.
```
notifyValidationChange = (isValid) => {
     if (this.props.onValidationChange) {
       this.props.onValidationChange(isValid);
     }
   }
```